### PR TITLE
Fix/microsalt pipeline name

### PR DESCRIPTION
**Pull request by barrystokman**
This PR adds/fixes ...

### How to prepare for test
- [ ] ssh to hasta (depending on type of change)
- [ ] install on stage:
`bash /home/proj/production/servers/resources/hasta.scilifelab.se/update-cg-stage.sh [THIS-BRANCH-NAME]`
- [ ] ssh to clinical-db (depending on type of change)
- [ ] install on stage:
`bash /home/proj/production/servers/resources/clinical-db.scilifelab.se/update-clinical-api-stage.sh [THIS-BRANCH-NAME]`

### How to test
- [ ] login to ...
- [ ] do ...

### Expected test outcome
- [ ] check that ...
- [ ] Take a screenshot and attach or copy/paste the output.

## Review
- [ ] code approved by
- [ ] tests executed by
- [ ] "Merge and deploy" approved by
Thanks for filling in who performed the code review and the test!

### This [version](https://semver.org/) is a
- [ ] **MAJOR** - when you make incompatible API changes
- [ ] **MINOR** - when you add functionality in a backwards compatible manner
- [ ] **PATCH** - when you make backwards compatible bug fixes or documentation/instructions

### CODE REVIEW

File: cg/constants.py [hunk 1]
```diff
@@ -290,6 +290,16 @@
     },
 }
 
+DATA_ANALYSIS_TO_PIPELINE = {
```
Comment (moonso) [inline cg/constants.py:293]: 👍

File: cg/meta/store/base.py [hunk 2]
```diff
@@ -229,3 +232,14 @@ def _determine_missing_tags(mandatory_tags: list, found_tags: list) -> tuple:
...
     return are_tags_missing, missing_tags
+
+
+def _convert_analysis_to_pipeline(case: models.Family) -> str:
+    """
+    convert data_analysis to pipeline name
+    """
+    data_analysis = case.links[0].sample.data_analysis
+    pipeline = DATA_ANALYSIS_TO_PIPELINE.get(data_analysis)
+    if not pipeline:
```
Comment (moonso) [inline cg/meta/store/base.py:243]: 👍

File: cg/meta/store/base.py [hunk 3]
```diff
@@ -229,3 +232,14 @@ def _determine_missing_tags(mandatory_tags: list, found_tags: list) -> tuple:
     are_tags_missing = bool(len(missing_tags) > 0)
 
     return are_tags_missing, missing_tags
+
+
+def _convert_analysis_to_pipeline(case: models.Family) -> str:
```
Comment (moonso) [inline cg/meta/store/base.py:237]: This was smart as it will be easy to change when analysis moves to case level 💯

**Review by moonso — APPROVED** [review]
Great 🌸

**Comment by barrystokman** [comment]
Closing, fixed in https://github.com/Clinical-Genomics/cg/pull/804 and https://github.com/Clinical-Genomics/cg/pull/805

**Pull request closed without merging**